nbs/explains/minidataapi.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
_Commented on notebook `nbs/explains/minidataapi.ipynb` Cell 15 Line 1_
```python
try:
```
I don't normally mind about formatting for code much, but for sample docs I care more -- so let's follow fast.ai style here and put the `try` body on the same line.